acoustid/data/stats.py

```
@@ -128,8 +139,9 @@
     raise ValueError("invalid lookup user agent stats key")
 
 
-def update_user_agent_counter(redis, application_id, user_agent, ip):
-    # type: (Any, int, str, str) -> None
+def update_user_agent_counter(
+    redis: Any, application_id: int, user_agent: str, ip: str
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `redis`

(ANN401)